git-lfs/PKGBUILD

```diff
@@ -0,0 +1,41 @@
+# Maintainer: Brendan Forster <brendan@github.com>
+
+pkgname=('git-lfs')
+pkgver=1.1.2
+pkgrel=1
+pkgdesc="An open source Git extension for versioning large files"
+arch=('i686' 'x86_64')
+url="https://github.com/github/git-lfs"
+license=('MIT')
+groups=('VCS')
+
+source=(
+      "https://github.com/github/git-lfs/releases/download/v$pkgver/git-lfs-windows-386-$pkgver.zip"
+      "https://github.com/github/git-lfs/releases/download/v$pkgver/git-lfs-windows-amd64-$pkgver.zip")
+
+md5sums=('8f4cc7008042fa88f9d10b257d93d632'
+         '7034f7ea80fb2d12033829a8d9d1c6c4')
+
+# both files have a git-lfs.exe so we need to extract these manually
+# TODO: perhaps this can be simplified into one expression
+noextract=("git-lfs-windows-386-$pkgver.zip" "git-lfs-windows-amd64-$pkgver.zip")
```

Comment:
By having one source defined using `case` the `noextract` becomes unnecessary. But thanks for the tip...